static/js/public/store-categories.js

```diff
@@ -10,10 +13,19 @@ function getCategory(holder) {
   const category = holder.dataset.category;
 
   // Write the html and init the carousel
-  const writeCategory = function(response) {
-    holder.innerHTML = response;
-    new Carousel(holder.querySelector('.p-carousel'));
-  };
+  const writeCategory = function() {
+    const _category = category;
```

Comment:
Why is this copy of `category` needed? Can't `category` used in places where `_category` is used now?

Reply:
Good catch, it's not in a look or anything so doesn't need to be a closure :+1: